--- setup.py.orig	2022-11-28 08:02:35.054301670 +1000
+++ setup.py	2022-11-28 08:02:52.799109689 +1000
@@ -91,7 +91,7 @@
             raise SystemExit("need to define %GDAL_HOME%")
         extraargs['include_dirs'] = [os.path.join(gdalhome, 'include')]
         extraargs['library_dirs'] = [os.path.join(gdalhome, 'lib')]
-        extraargs['libraries'] = ['gdal_i']
+        extraargs['libraries'] = ['gdal']
     else:
         # Unix - can do better with actual flags using gdal-config
         import subprocess
