--- setup.py.orig	2022-12-24 16:57:38.874855772 +1000
+++ setup.py	2022-12-24 16:57:47.746750657 +1000
@@ -45,12 +45,6 @@
 # don't build extensions if we are in readthedocs
 withExtensions = os.getenv('READTHEDOCS', default='False') != 'True'
 
-try:
-    from osgeo import gdal  # noqa
-except ImportError:
-    if withExtensions:
-        raise SystemExit("GDAL with Python bindings must be installed first")
-
 import tuiview
 
 
